web/wp-content/themes/dctx-v2/template-parts/scaffolding/scaffolding-blank.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-z7ggep.php, but found phpcs-scan-z7gGep.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).